jenkins_pipelines/environments/uyuni-prs-ci-tests

```diff
@@ -49,7 +49,7 @@ node('pull-request-test-nue') {
         additional_repo_url = params.additional_repo_url;
         terraform_parallelism = params.terraform_parallelism;
         sumaform_gitrepo = "https://github.com/uyuni-project/sumaform.git";
-        sumaform_ref = "master";
```

Comment:
are we still on the test branch?

Reply:
Yes, rename_cucumber_testsuite_module is the branch that has the cucumber testsuite module changes needed for the renaming. I will merge this branch and move the ref back to master once all the renaming is done. The old naming and new naming can't be supported with the same cucumber testsuite module. I need to differentiate as long as we have both naming conventions.